index.html

```diff
@@ -61,6 +61,8 @@
         }(document, 'script', 'facebook-jssdk'));
     </script>
 
+    <script src="res/vanilla.js"></script>
```

Comment:
Sorry to keep requesting changes, this is really good work 👍

I was just about to merge but noticed this addition. `vanilla.js` appears to be empty. Is it necessary? Any chance you could remove it please? Once that's done I'll approve and merge. Thank you so much! 😀